src/plugins/manus/install-dependencies.sh

```
@@ -0,0 +1,161 @@
+if [ ! -d "/var/local/git/grpc" ]; then
+    git clone --recurse-submodules -b v1.28.1 --depth 1 https://github.com/grpc/grpc /var/local/git/grpc
+else
+    echo "gRPC repository already exists, skipping clone"
+fi
+
+# Check if running on x86_64
+if [ "$(uname -m)" == "x86_64" ]; then
+    echo "[3/7] Building protobuf for x86_64..." && \
+    cd /var/local/git/grpc/third_party/protobuf && \
+    ./autogen.sh && ./configure --enable-shared && \
+    make -j$(nproc) && make -j$(nproc) check && make install && ldconfig
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)